src/lavinmq/amqp/channel_reply_code.cr

```diff
@@ -7,6 +7,7 @@ module LavinMQ
       NOT_FOUND           = 404
       RESOURCE_LOCKED     = 405
       PRECONDITION_FAILED = 406
+      NOT_ALLOWED         = 530
```

Comment:
I think "506 - resource-error" might be a better fit:
> The server could not complete the method because it lacked sufficient
> resources. This may be due to the client creating too many of some type
> of entity.

Reply:
Used 530 to follow Rabbit implementation. But I'll do 506, makes sense